be/src/exprs/vectorized/function_call_expr.cpp

```diff
@@ -124,7 +117,7 @@ ColumnPtr VectorizedFunctionCallExpr::evaluate(starrocks::ExprContext* context,
         args.emplace_back(column);
     }
 
-    if (_is_rand_function) {
+    if (returns_random_value()) {
```

Comment:
Better to use variable to avoid calling this function very chunk. This is bad to performance.